recipes/_install-windows.rb

```diff
@@ -24,17 +24,19 @@
 temp_file = ::File.join(Chef::Config[:file_cache_path], 'ddagent-cli.msi')
 
 # Download the installer to a temp location
-remote_file 'MSI installer' do
-  path temp_file
+remote_file temp_file do
   source node['datadog']['windows_agent_url'] + dd_agent_msi
+  checksum node['datadog']['windows_agent_checksum'] if node['datadog']['windows_agent_checksum']
```

Comment:
The `checksum` is used to prevent Chef from re-downloading the file (based on Chef's docs). And to prevent re-downloading the file this resource already uses the etag given in the response's headers. Do you have a case in mind where providing the checksum here gives better results?